metadata.json

```diff
@@ -14,7 +14,7 @@
     },
     {
       "name": "WhatsARanjit/node_manager",
-      "version_requirement": ">= 0.7.5 < 2.0.0"
+      "version_requirement": "0.7.5"
```

Comment:
Why don't you want to allow version 0.8.0?

Reply:
@bastelfreak We had customers with upgrade issues since the `WhatsARanjit/node_manager` module has been upgraded to 0.8.0. This is due to the fact that the semantics of updating rules changed when `purge_behavior` is `none`. Pinning to a working version is the quick fix, we will follow up with a strategic fix we are currently reviewing.

The issue is that the new behaviour doesn't permit node_manager to change a rule's initial `AND` into an `OR` which PEADM was supposed to do for a certain node group. As a result, the node group suddenly missed infrastructure nodes with upgrade issues as a result.

Reply:
Incidentally, we noticed that there were no new spec tests for the rule changes added, so this change flew "under the radar".

Reply:
Has puppet any plans to contribute a fix for node_manager? My impression is that most people ignore those depedencies and it's not documented in the README.md nor the peadm::install/peadm::upgrade examples.

Reply:
Hey @bastelfreak. We're bringing the node_manager module into the puppetlabs namespace and will be issuing a fix there. Then PEADM will be updated to use this instead of pinning.

Reply:
Hi @bastelfreak, how to install PEADM is documented here, for example: https://github.com/puppetlabs/puppetlabs-peadm/blob/main/documentation/install.md#bolt-3-usage .
This install procedure will actually use metadata's dependency information to pull in dependencies.

I'm very interested to hear what improvements to the installation docs/methods you can suggest. /cc @GSPatton